### ISSUE TITLE
[a11y] Modal close button is not accessible

### ISSUE THREAD

**Issue by FunctionDJ** [couds/react-bulma-components#385]
**Describe the bug**
The `<Modal.Card.Header>` by default shows a ".delete" button for closing the modal.
This button is missing a discernable text: https://dequeuniversity.com/rules/axe/4.4/button-name?application=axeAPI
Users relying on screenreaders will receive no information about the purpose or effect of this button.
